proto/pdpb.proto

```diff
@@ -29,6 +29,12 @@ service PD {
 
     rpc AllocID(AllocIDRequest) returns (AllocIDResponse) {}
 
+    rpc GetAllocID(GetAllocIDRequest) returns (GetAllocIDResponse) {}
```

Comment:
How about merging `GetAllocID` and `RecoverAllocID` to one function, like `EbsSnapshotRecover`?

Reply:
GetAllocID is for backup, RecoverAllocID for restore